arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -566,153 +572,2046 @@
+          "enum": [
+            "Active",
+            "Creating",
+            "Deleting",
+            "Disabled",
+            "ReceiveDisabled",
+            "Renaming",
+            "Restoring",
+            "SendDisabled",
+            "Unknown"
+          ],
+          "x-ms-enum": {
+            "name": "EntityStatus",
+            "modelAsString": false
+          }
+        },
+        "SubscriptionCount": {
+          "format": "int32",
+          "type": "integer",
+          "description": "Number of subscriptions."
+        },
+        "SupportOrdering": {
+          "type": "boolean",
+          "description": "Value that indicates whether the topic supports ordering."
+        },
+        "UpdatedAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The exact time the message has been updated."
+        },
+        "UserMetadata": {
+          "type": "string",
+          "description": "The user metadata."
+        }
+      },
+	  "x-ms-client-flatten": true,
+      "description": "Represents the metadata description of the topic."
+    },
+    "TopicResource": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "URL of the resource."
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the topic."
+        },
+        "type": {
+          "type": "string",
+          "description": "The name of the resource."
+        },
+        "location": {
+          "type": "string",
+          "description": "Supported Azure location in which the resource exists."
+        },
+        "properties": {
+          "$ref": "#/definitions/TopicProperties",
+          "description": "Properties of the Topics."
+        }
+      },
+
+      "description": "Description of topic Resource."
+    },
+    "TopicListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TopicResource"
+          },
+          "description": "Result of the List Topics operation."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to the next set of results. Not empty if Value contains incomplete list of Topics"
+        }
+      },
+      "description": "The response of the List Topics operation."
+    },
+    "QueueCreateOrUpdateParameters": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Queue name."
+        },
+        "type": {
+          "type": "string",
+          "description": "ARM type of the namespace."
+        },
+        "location": {
+          "type": "string",
+          "description": "location of the resource."
+        },
+        "properties": {
+          "$ref": "#/definitions/QueueProperties",
+          "description": "properties of the Queues."
```

Comment:
as I said in one of my previous comments, that this extension should be applied on the ref. Please go through that.